akka-stream-tests/src/test/scala/akka/stream/impl/fusing/InterpreterSpec.scala

```diff
@@ -447,42 +484,330 @@ class InterpreterSpec extends InterpreterSpecKit {
 
     }
 
-    "handle failure inside op" in new TestSetup(Seq(Map((x: Int) ⇒ if (x == 0) throw TE else x))) {
+    "emit failure when op throws" in new TestSetup(Seq(Map((x: Int) ⇒ if (x == 0) throw TE else x, stoppingDecider))) {
```

Comment:
maybe these supervision tests should be moved to another Spec, this file starts to grow very large